### DISCUSSION TITLE
Projects to work on at Tech Club

### DISCUSSION THREAD

**Discussion by viett-appcard** [AppCardTechClub/.github#1] [General]
Let's gather some ideas on future agenda for tech club
@mgayevskyappcard @dgalloappcard @Burgoa01 @yotamappcard 

react emojis to ideas you like

**Comment by viett-appcard** [comment]
- A buzzer for #prod-alerts
  - Whenever echobot make a message without the word "OK" , buzzer with sound
  - Whenever @here in a message, buzzer with sound
  - guide, buy the parts

**Comment by mgayevskyappcard** [comment]
- An auto data populating powerpoint for campaign analyses and insights reports.
